### ISSUE TITLE
About the dataset and Model details

### ISSUE THREAD

**Issue by xiaofengShi** [ad71/Flipkart-Grid#1]
Forgive me for my bad English. I want to ask you a few questions. A picture contains only one box in your training dataset and inference situation? Don't you calculate the object confidence?  I didn't see the relevant details in your blog, and I noticed 

> So I used standard architectures, replaced the last layer with a dense layer with four outputs (x1, y1 and x2, y2 being co-ordinates of the top left corner and the bottom right corner), removed activations and used a classical regression loss.
in you blog.

**Comment by ad71** [comment]
Yes, a picture contains only one box both for training and inference. I think you missed the point that the problem didn't require us to classify the images. Just the bounding box for the dominant image was to be found out. A confidence score is therefore pointless. This is just a subproblem of the standard object-detection problem. I'm sorry if it was misleading.

**Comment by xiaofengShi** [comment]
@ad71  Thank you very much for your reply. Now that I understand the task of your model, your research on the regression loss calculation of bbox in the blog is very worth learning.

**Comment by ad71** [comment]
@xiaofengShi good to hear!
